apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -146,19 +153,34 @@ export default class AnimationPickerBody extends React.Component {
     if (this.props.hideBackgrounds) {
       categories = categories.filter(category => category !== 'backgrounds');
     }
+    // Level-specific animations currently have the following categories:
+    // "level_animations", "backgrounds", "animals".
```

Comment:
Should we make an enum/constants for this?

Reply:
Opted for constants because 'backgrounds' is used in other contexts.